packages/ChangesReloaded-Core.package/CRBrowser.class/instance/updateAfterHistorySelection.st

```diff
@@ -2,4 +2,5 @@ updating
 updateAfterHistorySelection
 
 	self refreshChangeListItems.
+	self repaintChangeListSelections.
```

Comment:
This only repaints _one_ selection, doesn't it? One could argue that the previously selected change gets repainted to not being selected and the new selected one to be selected, but then we should rename `repaintHistoryListSelection` to `repaintHistoryListSelections`, too, because that's actually the same case. (Or we rename `repaintChangeListSelections` to `repaintChangeListSelection`)

Reply:
Nahh, this is just legacy (from the multi selection list). Good you spotted that, we changed it to Singular (since now, it is a single selection list) ^^